### ISSUE TITLE
Question about validation set pairing and evaluation metrics

### ISSUE THREAD

**Issue by whitescilence** [MRIxFields/MRIxFields2026#2]
Hi, thank you for providing the MRIxFields2026 baseline code.

I have a question about the validation/evaluation process.

As I understand it, the training data for CycleGAN is unpaired. However, I am not sure whether the validation set is paired between the source field strength and the target field strength.

If the validation set is also unpaired, how should we calculate quantitative evaluation metrics such as NRMSE, SSIM, and LPIPS? These metrics usually require the generated image and the corresponding ground-truth image to be spatially aligned.

Could you please clarify:

Is the validation set paired or unpaired?
If it is paired, how can we identify the corresponding source and target images/subjects?
If it is unpaired, what evaluation metrics or evaluation protocol should we use?

<img width="469" height="581" alt="Image" src="https://github.com/user-attachments/assets/1292189b-f1bd-4d41-91b7-0660d037ff38" />
I have attached an example of the validation data structure below.

Thank you!
